modules/control/controller/mpc_controller.cc

```diff
@@ -372,41 +373,57 @@ Status MPCController::ComputeControlCommand(
   double control_gain_truncation_ratio = 0.0;
   double unconstraint_control = 0.0;
   const double v = VehicleStateProvider::Instance()->linear_velocity();
-  if (!common::math::SolveLinearMPC(matrix_ad_, matrix_bd_, matrix_cd_,
-                                    matrix_q_updated_, matrix_r_updated_,
-                                    lower_bound, upper_bound, matrix_state_,
-                                    reference, mpc_eps_, mpc_max_iteration_,
-                                    &control, &control_gain, &addition_gain)) {
-    AERROR << "MPC solver failed";
+
+  std::vector<double> *control_cmd;
+  if (FLAGS_use_osqp_solver) {
+    apollo::common::math::MpcOsqp mpc_osqp(
+        matrix_ad_, matrix_bd_, matrix_q_updated_, matrix_r_updated_,
+        lower_bound, upper_bound, matrix_state_, mpc_max_iteration_);
+    if (!mpc_osqp.MpcOsqpSolver(control_cmd)) {
+      AERROR << "MPC OSQP solver failed";
+    } else {
+      AINFO << "MPC OSQP problem solved! ";
```

Comment:
Use ADEBUG instead of AINFO to reduce unnecessary logging.

Reply:
Fixed. Thank you!